src/features/pastelAddressBlock/PastelAddressBlock.tsx

```diff
@@ -0,0 +1,292 @@
+          <div className={[styles.verticalflex, styles.marginleft].join(' ')}>
+            {label && (
+              <div className={styles.margintoplarge}>
+                <div className={[styles.sublight].join(' ')}>Label</div>
+                <div
+                  className={[styles.padtopsmall, styles.fixedfont].join(' ')}
+                >
+                  {label}
+                </div>
+              </div>
+            )}
+
+            <div className={[styles.sublight, styles.margintopSmall].join(' ')}>
+              Funds
+            </div>
+            <div className={[styles.padtopsmall].join(' ')}>
+              {currencyName} {balance}
+            </div>
+            <div className={[styles.padtopsmall].join(' ')}>
+              {Utils.getPslToUsdString(pslPrice, balance)}
+            </div>
+            <div
+              className={[styles.margintoplarge, styles.breakword].join(' ')}
+            >
+              {privateKey && (
+                <div>
+                  <div className={[styles.sublight].join(' ')}>Private Key</div>
+                  <div
+                    className={[
+                      styles.breakword,
+                      styles.padtopsmall,
+                      styles.fixedfont,
+                    ].join(' ')}
+                    style={{
+                      maxWidth: '600px',
+                    }}
+                  >
+                    {privateKey}
+                    <div
+                      className={[styles.margintoplarge, styles.highlight].join(
+                        ' ',
+                      )}
+                    >
+                      <i
+                        className={[
+                          styles.yellow,
+                          styles.padrightsmall,
+                          styles.small,
+                          'fas',
+                          'fa-exclamation-triangle',
+                        ].join(' ')}
+                      />
+                      WARNING: DO NOT SEND TO ANYONE
+                    </div>
+                  </div>
+                </div>
+              )}
+            </div>
+
+            <div
+              className={[styles.margintoplarge, styles.breakword].join(' ')}
+            >
+              {viewKey && (
+                <div>
+                  <div className={[styles.sublight].join(' ')}>Viewing Key</div>
+                  <div
+                    className={[
+                      styles.breakword,
+                      styles.padtopsmall,
+                      styles.fixedfont,
+                    ].join(' ')}
+                    style={{
+                      maxWidth: '600px',
+                    }}
+                  >
+                    {viewKey}
+                  </div>
+                </div>
+              )}
+            </div>
+
+            <div>
+              <button
+                className={[styles.primarybutton, styles.margintoplarge].join(
```

Comment:
We can use `classnames` package for this, it was recently added. Please see https://github.com/pastelnetwork/pastel-electron-wallet/pull/20/files (and change everywhere in this PR)

Reply:
I have updated